src/lib/coaler/embedder/ConformerEmbedder.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <GraphMol/ROMol.h>
+
+namespace coaler::embedder {
+
+    using CoreAtomMapping = std::map<int, RDGeom::Point3D>;
+
+    class ConformerEmbedder {
+      public:
+        // ConformerEmbedder(const RDKit::ROMol& core,
+        //                   unsigned numCoreConfigs);
+
+        ConformerEmbedder(RDKit::ROMol core, const CoreAtomMapping& coreMap);
+
+        bool embedWithFixedCore(RDKit::ROMol& mol, unsigned numConfs);
+
+      private:
+        RDKit::ROMol m_core;
```

Comment:
I would suggest to rename member variables from m_core to core_ like in http://micro-os-plus.github.io/develop/naming-conventions/

Reply:
Hm I have the same preference as Chris but we could discuss this in the group

Reply:
We'd have to do that for the entire project. (entire multi-align lib uses the m_name convention rn). I'd suggest we leave that for a generalized refactoring.